src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,30 @@
+package duke.task;
+
+public abstract class Task {
+    protected String task;
+    protected boolean done;
```

Comment:
Boolean name should sound like booleans. eg isDone

Reply:
Oops